AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/RemoteSyncEngine.swift

```diff
@@ -231,10 +249,20 @@ class RemoteSyncEngine: RemoteSyncEngineBehavior {
         stateMachine.notify(action: .activatedMutationQueue)
     }
 
+    private func cleanup(error: AmplifyError?) {
+        // TODO:
+        // handle clean up here
```

Comment:
nit: put the message on the same line of the TODO so IDE displays the warning accordingly

Reply:
Ah! I didn't know that.  I will do that in the future, but this TODO has already been implemented in a subsequent PR.